src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -1456,6 +1456,12 @@
 				});
 			}
 
+			prototype.getDividers = function () {
+				return this._items
+					.filter(function (elem) {
+						return elem.classList.contains("ui-listview-divider");
```

Comment:
change "ui-listview-divider" to classes.DIVIDER

Reply:
done